Monika After Story/game/script-compliments.rpy

```diff
@@ -530,3 +530,67 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...And I'll be forever grateful that you did~"
     return
+    
+    init 5 python:
+        addEvent(
+            Event(
+                persistent._mas_compliments_database,
+                eventlabel="mas_compliment_you",
+                prompt="... I'm always thinking about you!",
+                unlocked=True
+            ),
+            eventdb=store.mas_compliments.compliment_database
+        )
+
+label mas_compliment_you:
+    if not renpy.seen_label("mas_compliment_you_2"):
+        call mas_compliment_you_2
+    else:
+        call mas_compliment_you_3
+    return
+
+label mas_compliment_you_2:
+    m 1wubso "..."
+    m 1wubfb "[player]..."
+    
+    if mas_isMoniUpset(higher=True):
```

Comment:
I think the intended compliment should start at normal and upset should be given its own path.

Reply:
Like, a whole new topic? I am unsure what you mean.

Reply:
Like what you have with the lines below with mas_isMoniDis and mas_isMoniBroken. I mean add one exclusively for mas_isMoniUpset and have this be normal+.

Reply:
Oh! 

Um, The "Upset" bit is meant mainly for everyone, with the lowest level for it showing up being her upset. (Upset Moni seems close to standard Moni, but a little less optimistic in most of the topics I've read)

I felt that having the Touched response would be weird if Moni was sad or upset, So I figured we could omit anything besides the opening acknowledgement for the sad bits. 

Unless I am completely misunderstanding.

Reply:
Ah, never mind, misunderstood.